### PR TITLE
Fix wording

### PR DESCRIPTION
**Pull request by Mx-Glitter**

### CODE REVIEW

**Comment by Mx-Glitter** [comment]
@tychota @yleflour I'm blocked by dangerfile, can you help me? 😇
image

File: react-native/setup/setup-and-deploy-new-project-to-staging.mo.md [hunk 1]
```diff
@@ -117,4 +112,4 @@ bundle exec fastlane android deploy --env=staging
 
 ## Troubleshooting
 
-If 'Cloning GitHub repo' takes more than 2 minutes: the github servers may be untrusted Trigering a `git clone git@github.com:bamlab/certificates.git` will fix it.
+If 'Cloning GitHub repo' takes more than 2 minutes: the github servers may be untrusted Triggering a `git clone git@github.com:bamlab/certificates.git` will fix it.
```
Comment (louiszawadzki) [inline react-native/setup/setup-and-deploy-new-project-to-staging.mo.md]: missing '.':

If 'Cloning GitHub repo' takes more than 2 minutes: the github servers may be untrusted.
Triggering a `git clone git@github.com:bamlab/certificates.git` will fix it.

Comment (Mx-Glitter) [reply react-native/setup/setup-and-deploy-new-project-to-staging.mo.md]: Fixed with last commit.

**Comment by tychota** [comment]
Hello @Triiistan 

First thank you for contribution 🎉 

---

Short explanation:
- danger is run on the CI
- a valid github token is passed to danger so danger has a lot of right (not admin but still right to mess thinks up)
- everyone can do a PR and modify `dangerfile.js``

Thus, I authorised the access to @bamlab members only, to prevent a security issue.

---

Your typo fix does not need to trigger the poka yoke anyway so I will merge like this :)

**Pull request merged**